packages/manager/modules/bm-server-components/src/technical-details/technical-details.controller.js

```diff
@@ -35,7 +39,10 @@ export default class TechnicalDetailsController {
     const freqUnit = this.$translate.instant(
       'dedicated_server_dashboard_technical_details_ram_frequency_unit',
     );
-    const ramSize = ram.size ? `${ram.size} GB` : '';
+    const gbTranslated = this.$translate.instant(
+      'dedicated_server_dashboard_technical_details_ram_size_unit',
+    );
+    const ramSize = ram.size ? `${ram.size} ${gbTranslated}` : '';
```

Comment:
Shouldn't the size be handled in the translation as a parameter ?  It might be easier and more suitable if there are other formattings in other languages

Reply:
Ideally yes. But the objective of this work is to extract it as a reusable component with minimal impact and changes. I did not do lot of changes for the existing code.